src/core/quantum_gate.jl

```diff
@@ -1474,13 +1474,13 @@ phase_shift(phi,T::Type{<:Complex}=ComplexF64) = DiagonalOperator(T[1.,exp(im*ph
     universal(theta, phi, lambda)
 
 Return the `Operator` which performs a rotation about the angles `theta`, `phi`, and `lambda`.
-See: https://qiskit.org/textbook/ch-states/single-qubit-gates.html#generalU
+See: Theorem 4.1 in [Quantum Computation and Quantum Information by Nielsen and Chuang](http://mjg.global.s3.amazonaws.com/static/projects/refs/Quantum_Computation_and_Quantum_Information.pdf).
```

Comment:
I think we should rather link to the Wikipedia page

Reply:
do you mean [this](https://en.wikipedia.org/wiki/Quantum_logic_gate) page? Unlike the textbook, this page does not mention the U-gate specifically

Reply:
changed